tests/README.rst

```diff
@@ -85,3 +89,28 @@ If you want to contribute towards adding a new tests in Fledge, make sure you fo
 
 - Test file name should begin with the word ``test_`` to enable pytest auto test discovery.
 - Make sure you are placing your test file in the correct test directory. For example, if you are writing a unit test, it should be located under ``$FLEDGE_ROOT/tests/unit/python/fledge/<component>`` where component is the name of the component for which you are writing the unit tests. For more information of type of test, refer to the test categories.
+
+Code Coverage
+-------------
+
+Python Tests
+++++++++++++
+
+Fledge uses PyTest-cov Framework of Pytest as the code coverage measuring tool for python tests, For more information on PyTest-cov please refer to |pytest-cov docs|.
+
+To install PyTest-cov Framework along with Pytest Framework use the following command:
+::
+   python3 -m pip install pytest==3.6.4 pytest-cov==2.9.0
```

Comment:
Why pytest 3.6.4? It's a very old and we should find the latest stable version with our requirements.
May be `6.2.5` if min python 3.6 is required

Reply:
ok, I'll find the latest stable version with our requirements and update you.

Reply:
Deepanshu had logged an issue with Pytest 4+.
In the scope of this PR, upgrade is not required. 

We can work on that when that Jira is handled and find/replace all.